src/main/java/com/nooul/apihelpers/springbootrest/services/FilterService.java

```diff
@@ -31,144 +38,159 @@
     @Autowired
     private CustomSpecifications<T> specifications;
 
-
-    public long countBy(QueryParamWrapper queryParamWrapper, BaseRepository<T, I> repo) {
```

Comment:
I need this helper for counts not sure why you removed it..

Reply:
I understand this requirments, which I do not have in my projects.